### ISSUE TITLE
zlib_util_c.h:2:18: fatal error: zlib.h: No such file or directory

### ISSUE THREAD

**Issue by YetAnotherRandomCoder** [go-vgo/robotgo#240]
- Robotgo version (or commit ref): newest
- Go version: 1.11
- Gcc version: 5.1.0
- Operating system and bit: Windows 10
- Resolution:
- Can you reproduce the bug
  - [x ] Yes (provide example code)
  - [ ] No
  - [ ] Not relevant

Every time I want to try robotgo I get the following error:

# github.com/go-vgo/robotgo
In file included from ./bitmap/../base/str_io_c.h:2:0,
                 from ./bitmap/goBitmap.h:17,
                 from ..\go\src\github.com\go-vgo\robotgo\robotgo.go:45:
./bitmap/../base/zlib_util_c.h:2:18: fatal error: zlib.h: No such file or directory
compilation terminated.
exit status 2
Process exiting with code: 1

I have already found the similar issue here: https://github.com/go-vgo/robotgo/issues/100

But their solution to use MingW is not working for me or am I doing something horrible wrong here? Any ideas?

my gcc config:

C:\Users\Alf>gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=C:/TDM-GCC-64/bin/../libexec/gcc/x86_64-w64-mingw32/5.1.0/lto-wrapper.exe
Target: x86_64-w64-mingw32
Configured with: ../../../src/gcc-5.1.0/configure --build=x86_64-w64-mingw32 --enable-targets=all --enable-languages=ada,c,c++,fortran,lto,objc,obj-c++ --enable-libgomp --enable-lto --enable-graphite --enable-cxx-flags=-DWINPTHREAD_STATIC --disable-build-with-cxx --disable-build-poststage1-with-cxx --enable-libstdcxx-debug --enable-threads=posix --enable-version-specific-runtime-libs --enable-fully-dynamic-string --enable-libstdcxx-threads --enable-libstdcxx-time --with-gnu-ld --disable-werror --disable-nls --disable-win32-registry --prefix=/mingw64tdm --with-local-prefix=/mingw64tdm --with-pkgversion=tdm64-1 --with-bugurl=http://tdm-gcc.tdragon.net/bugs
Thread model: posix
gcc version 5.1.0 (tdm64-1)

- Provide example code:

package main

import (
	"github.com/go-vgo/robotgo"
)

func main() {
	robotgo.ScrollMouse(10, "up")
	robotgo.MouseClick("left", true)
	robotgo.MoveMouseSmooth(100, 200, 1.0, 100.0)
}

**Comment by 0xbkt** [comment]
I may have fixed this in Go 1.13.

```fish
env GOOS=windows \
       GOARCH=amd64 \
       CGO_ENABLED=1 \
       CC=x86_64-w64-mingw32-gcc-posix \
       CXX=x86_64-w64-mingw32-g++-posix \
       CGO_CFLAGS="-g -O2 -I/usr/local/x86_64-w64-mingw32/include" \
       CGO_LDFLAGS="-g -O2 -L/usr/local/x86_64-w64-mingw32/lib -L/usr/local/x86_64-w64-mingw32/lib" \
       go build -trimpath -ldflags="-w -s -extldflags -static" -x
```

**Comment by vcaesar** [comment]
All code tests and C dependencies use MinGw-64, TDM-GCC has not been tested.
